recipes/catch2/2.x.x/conanfile.py

```diff
@@ -44,6 +46,9 @@ def configure(self):
         if not self.options.with_main:
             del self.options.fPIC
             del self.options.with_benchmark
+        if self.options.default_reporter:
+            if '"' not in str(self.options.default_reporter):
```

Comment:
Why is it so important that the default reporter is surrounded by double quotes?
I think conan's cmake helper will automatically escape variables with spaces.

Reply:
Because this is how this macro is used: https://github.com/catchorg/Catch2/blob/216713a4066b79d9803d374f261ccb30c0fb451f/include/internal/catch_config.hpp#L66
It took me quite some time to discover, and I see problems with escaping at least in `compile_commands.json`, in the command line displayed with `ps` and probably with conan `cmake` build helper, not sure about the last one. Anyway, I am planning on submitting a PR in catch that would allow to pass unquoted string.